repos/cli.proj

```diff
@@ -49,11 +49,10 @@
       Include="
         Microsoft.AspNetCore.All;
         Microsoft.AspNetCore.App"
-      Version="2.1.2" />
-    <!-- This package wasn't built in latest 2.1.2 build: stick with what we had from 2.1.1. -->
+      Version="2.1.3" />
     <ExtraPackageVersionPropsPackageInfo
       Include="Microsoft.NET.Sdk.Razor"
-      Version="2.1.1" />
+      Version="2.1.2" />
```

Comment:
See https://github.com/dotnet/source-build/issues/611 for plans to auto-update this, and why 2.1.2 was built in the 2.1.3 build train.